dll/win32/comctl32/rebar.c

```diff
@@ -1713,13 +1713,6 @@ REBAR_CommonSetupBand(HWND hwnd, const REBARBANDINFOW *lprbbi, REBAR_BAND *lpBan
 
     lpBand->fMask |= lprbbi->fMask;
 
-    if( (lprbbi->fMask & RBBIM_STYLE) &&
```

Comment:
This file is winesynced,
we usually surround our changes with `#ifdef __REACTOS__` in those until it's upstreamed.

Reply:
ok, then i would update it to include the ifdef

Reply:
Done